packages/ui-react/src/components/FeaturedShelf/FeaturedMetadata.tsx

```diff
@@ -0,0 +1,59 @@
+import type { PlaylistItem } from '@jwp/ott-common/types/playlist';
+import classNames from 'classnames';
+import type { CSSProperties } from 'react';
+import ChevronRight from '@jwp/ott-theme/assets/icons/chevron_right.svg?react';
+import { mediaURL } from '@jwp/ott-common/src/utils/urlFormatting';
+import { useNavigate } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import { testId } from '@jwp/ott-common/src/utils/common';
+
+import TruncatedText from '../TruncatedText/TruncatedText';
+import StartWatchingButton from '../../containers/StartWatchingButton/StartWatchingButton';
+import Button from '../Button/Button';
+import Icon from '../Icon/Icon';
+
+import styles from './FeaturedShelf.module.scss';
+
+const FeaturedMetadata = ({
+  item,
+  loading,
+  playlistId,
+  style,
+  hidden,
+}: {
+  item: PlaylistItem | null;
+  loading: boolean;
+  playlistId: string | undefined;
+  style?: CSSProperties;
+  hidden?: boolean;
+}) => {
+  const navigate = useNavigate();
+  const { t } = useTranslation('common');
+
+  if (!item) return null;
+
+  const hasVideo = item.sources.find((source) => source.file.indexOf('.m3u8') > -1 || source.file.indexOf('.mp4') > -1);
```

Comment:
We might need a util for this and also check for the content type. Some non-playable content types may contain dummy streams in the source list.

Reply:
I think we only want `movie` and `episode`?

Reply:
I see that `movie` is not a CONTENT_TYPE, so I just added an opt-out for `page`, `livechannel` and `hub` for now.